api/v1/server/headers/middleware.go

```diff
@@ -0,0 +1,24 @@
+package headers
+
+import (
+	"github.com/labstack/echo/v4"
+
+	"github.com/hatchet-dev/hatchet/api/v1/server/middleware"
+)
+
+func Middleware() middleware.MiddlewareFunc {
+	return func(r *middleware.RouteInfo) echo.HandlerFunc {
+		return func(c echo.Context) error {
+			c.Response().Header().Set("Strict-Transport-Security", "max-age=776000; includeSubDomains;  preload")
```

Comment:
For each of these headers, could you document the behavior that we're allowing and restricting?